canonical-websites/insights.ubuntu.com
# Update the footer

## Done

- Updated the year, legal link names
- Updated to the vanilla footer pattern
- Added the canonical logo

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- See that the 'footer wording to be amended from "Privacy policy" to Data privacy"'

## Issue / Card

Fixes #504

## Screenshots
